readthedocs/projects/forms.py

```diff
@@ -682,6 +714,8 @@ class Meta:
         labels = {
             "enabled": _("Enable Addons"),
             "doc_diff_enabled": _("Visual diff enabled"),
+            "filetreediff_enabled": _("Enabled"),
+            "filetreediff_ignored_files": _("Ignored files"),
```

Comment:
All these labels should be in the model itself, not just in the form.

Reply:
What we would need to use there, `verbose_name`?

Reply:
yeah

Reply:
👍🏼 -- I created an issue at https://github.com/readthedocs/readthedocs.org/issues/11986